lib/msal-browser/src/error/BrowserAuthErrorCodes.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export const pkceNotGenerated = "pkce_not_created";
+export const cryptoDoesNotExist = "crypto_nonexistent";
+export const emptyNavigateUriError = "empty_navigate_uri";
+export const hashEmptyError = "hash_empty_error";
+export const hashDoesNotContainStateError = "no_state_in_hash";
```

Comment:
suggest matching const names to the text:
`export const noStateInHash = "no_state_in_hash"`
There are quite a few of these in the list.  We likely can't alter things like "no_state_in_hash" as folks might worry if there's a change in error characteristics from one release to another.

Reply:
Great suggestion thanks! Done.